tests/functional/test_install.py

```diff
@@ -76,21 +79,19 @@ def test_editable_install(script):
     assert not result.files_updated
 
 
-@pytest.mark.network
-def test_install_editable_from_svn(script, tmpdir):
+def test_install_editable_from_svn(script, data):
     """
     Test checking out from svn.
     """
+    checkout_path = _create_test_package(script, vcs='svn')
     result = script.pip(
         'install',
-        '-e',
-        '%s#egg=initools-dev' %
-        local_checkout(
-            'svn+http://svn.colorstudy.com/INITools/trunk',
-            tmpdir.join("cache")
-        )
+        '-f', data.packages,
+        '-e', path_to_url(checkout_path) + '#egg=version-pkg'
     )
-    result.assert_installed('INITools', with_files=['.svn'])
```

Comment:
Hmmm. I think I removed this because for some reason, `.svn` wasn't showing up in `self.files_created`. I didn't know if this was expected or a bug:

```python
(Pdb++) pp self.files_created
{u'scratch/pip-test-package/version_pkg.egg-info': <FoundDir /private/var/folders/gw/w0clrs515zx9x_55zgtpv4mm0000gp/T/pytest-158/test_install_editable_from_svn0/workspace:scratch/pip-test-package/version_pkg.egg-info>,
 u'scratch/pip-test-package/version_pkg.egg-info/PKG-INFO': <FoundFile /private/var/folders/gw/w0clrs515zx9x_55zgtpv4mm0000gp/T/pytest-158/test_install_editable_from_svn0/workspace:scratch/pip-test-package/version_pkg.egg-info/PKG-INFO>,
 u'scratch/pip-test-package/version_pkg.egg-info/SOURCES.txt': <FoundFile /private/var/folders/gw/w0clrs515zx9x_55zgtpv4mm0000gp/T/pytest-158/test_install_editable_from_svn0/workspace:scratch/pip-test-package/version_pkg.egg-info/SOURCES.txt>,
 u'scratch/pip-test-package/version_pkg.egg-info/dependency_links.txt': <FoundFile /private/var/folders/gw/w0clrs515zx9x_55zgtpv4mm0000gp/T/pytest-158/test_install_editable_from_svn0/workspace:scratch/pip-test-package/version_pkg.egg-info/dependency_links.txt>,
 u'scratch/pip-test-package/version_pkg.egg-info/entry_points.txt': <FoundFile /private/var/folders/gw/w0clrs515zx9x_55zgtpv4mm0000gp/T/pytest-158/test_install_editable_from_svn0/workspace:scratch/pip-test-package/version_pkg.egg-info/entry_points.txt>,
 u'scratch/pip-test-package/version_pkg.egg-info/top_level.txt': <FoundFile /private/var/folders/gw/w0clrs515zx9x_55zgtpv4mm0000gp/T/pytest-158/test_install_editable_from_svn0/workspace:scratch/pip-test-package/version_pkg.egg-info/top_level.txt>,
 u'venv/bin/version_pkg': <FoundFile /private/var/folders/gw/w0clrs515zx9x_55zgtpv4mm0000gp/T/pytest-158/test_install_editable_from_svn0/workspace:venv/bin/version_pkg>,
 u'venv/lib/python2.7/site-packages/version-pkg.egg-link': <FoundFile /private/var/folders/gw/w0clrs515zx9x_55zgtpv4mm0000gp/T/pytest-158/test_install_editable_from_svn0/workspace:venv/lib/python2.7/site-packages/version-pkg.egg-link>}
```